File: p2p/src/main/java/com/androdevlinux/percy/p2p/common/P2PDeviceService.java
package com.androdevlinux.percy.p2p.common;


import android.net.wifi.p2p.WifiP2pDevice;

import java.util.Map;

public class P2PDeviceService extends P2PDevice {


    private Map<String, String> txtRecordMap;

    public P2PDeviceService(WifiP2pDevice wifiP2pDevice) {
        super(wifiP2pDevice);
    }

    public Map<String, String> getTxtRecordMap() {
        return txtRecordMap;
    }

    public void setTxtRecordMap(Map<String, String> txtRecordMap) {
        this.txtRecordMap = txtRecordMap;
    }

}

File: p2p/src/main/java/com/androdevlinux/percy/p2p/common/P2PDeviceService.kt
package com.androdevlinux.percy.p2p.common


import android.net.wifi.p2p.WifiP2pDevice

class P2PDeviceService(wifiP2pDevice: WifiP2pDevice) : P2PDevice(wifiP2pDevice) {
    var txtRecordMap: Map<String, String>? = null
}
